BackofficeBundle/Manager/RedirectionManager.php

```diff
@@ -68,4 +73,18 @@ public function createRedirection($pattern, $nodeId, $language)
             }
         }
     }
+
+    /**
+     * @param string $nodeId
+     * @param string $language
+     */
+    public function deleteRedirection($nodeId, $language)
+    {
+        $redirections = $this->redirectionRepository->findByNode($nodeId, $language);
+        foreach ($redirections as $redirection) {
+            $this->documentManager->remove($redirection);
+            $this->documentManager->flush($redirection);
+            $this->eventDispatcher->dispatch(RedirectionEvents::REDIRECTION_DELETE, new RedirectionEvent($redirection));
```

Comment:
I didn't see a listener on that event, is there one ?
Without that, it would be far more efficient to directly remove the document from the base instead of finding matching documents, then hydrate them, then flush to remove them.